crates/trigger/src/cli.rs

```diff
@@ -22,8 +22,8 @@ pub const RUNTIME_CONFIG_FILE: &str = "RUNTIME_CONFIG_FILE";
 
 // Set by `spin up`
 pub const SPIN_LOCKED_URL: &str = "SPIN_LOCKED_URL";
+pub const SPIN_LOCAL_APP_DIR: &str = "SPIN_LOCAL_APP_DIR";
```

Comment:
This refers now to the application directory not `<app-dir>/.spin/` anymore right? Just want to confirm.

Reply:
Yep.